### PR TITLE
Stock routes for one month and one year prices

### PR DESCRIPTION
**Pull request by joeycor92**
Here I have added one month and one year stock routes which return the information to the front end in the form:
{
    "2022-09-13": "153.8400",
    "2022-09-14": "155.3100",
    "2022-09-15": "152.3700",
    "2022-09-16": "150.7000",
    "2022-09-19": "154.4800",
    "2022-09-20": "156.9000",
    "2022-09-21": "153.7200",
    "2022-09-22": "152.7400",
    "2022-09-23": "150.4300",
    "2022-09-26": "150.7700",
    "2022-09-27": "151.7600",
    "2022-09-28": "149.8400",
    "2022-09-29": "142.4800",
    "2022-09-30": "138.2000",
    "2022-10-03": "142.4500",
    "2022-10-04": "146.1000",
    "2022-10-05": "146.4000",
    "2022-10-06": "145.4300",
    "2022-10-07": "140.0900",
    "2022-10-10": "140.4200",
    "2022-10-11": "138.9800"
}

Make sure to run this on your own system before approving this (using git fetch and then git checkout -b whateveryouwanttocallit origin/313-Update-Routes-Historical-Data

Run npm i, as I installed 'moment' which is used.

I've added loads of comments to explain how it works.

### CODE REVIEW

**Review by DaraghK93 — COMMENTED** [review]
Screenshot 2022-10-13 at 11 33 16

Should this be in the Backend?

File: amplify/backend/function/expressServer/src/controllers/stockController.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
+const { mockComponent } = require('react-dom/test-utils');
```
Comment (caolan-power) [inline amplify/backend/function/expressServer/src/controllers/stockController.js:1]: Just pointing this out

**Review by caolan-power — APPROVED** [review]
could you include package.json so we have the dependencies too without needing a manual install? 

The routes work for me as you planned. Something to note was what we talked about yesterday. We need to get that data updating daily from now as already yesterdays data isn't in the db. Something for next week

This code working as planned though so looks good to me.

**Comment by joeycor92** [comment]
> could you include package.json so we have the dependencies too without needing a manual install?
> 
> The routes work for me as you planned. Something to note was what we talked about yesterday. We need to get that data updating daily from now as already yesterdays data isn't in the db. Something for next week
> 
> This code working as planned though so looks good to me.

Thanks, had pushed from the wrong folder so package.json wasn't included but is now. 

As regards the daily prices, agreed. My plan is to get the web scraping going either today or tomorrow, but before running that I'll be making sure that the daily prices for the missing days are included.

Thanks

**Review by caolan-power — APPROVED** [review]
Cool

**Review by DaraghK93 — APPROVED** [review]
looks great. That moment Library saves a lot of time I'd say. tested the routes there and all working.

**Pull request merged**